swift/BUILD.bazel

```diff
@@ -39,9 +39,27 @@ pkg_files(
     prefix = "tools/" + codeql_platform,
 )
 
+alias(
+    name = "swift-test-sdk",
+    actual = select({
+        "@bazel_tools//src/conditions:%s" % arch: "@swift_prebuilt_%s//:swift-test-sdk" % arch
+        for arch in ("linux", "darwin_x86_64", "darwin_arm64")
+    }),
+)
+
+pkg_files(
+    name = "swift-test-sdk-arch",
+    srcs = [":swift-test-sdk"],
+    prefix = "qltest/" + codeql_platform,
```

Comment:
is this a copy-paste error? (namely, the `qltest` part)

Reply:
That's on purpose. This idea is inherited from C++ support, though I cannot see any other usage of `qltest` in other language packs.
@criemen do you have a better place in mind? I'm happy to change/move this part if needed.

Reply:
I misunderstood what the PR is doing. I cleared that up now, this looks good to me 👍